### PR TITLE
Fix pyre error for logits

### PR DESCRIPTION
**Pull request by iseeyuan**
Summary: Fix the pyre error when logits is not initialized

Reviewed By: swolchok

Differential Revision: D70183946

### CODE REVIEW

**Review by jackzhxng — APPROVED** [review]
Trusting @swolchok's internal review
